### PR TITLE
Update Getting Started w/ valid username

### PR DESCRIPTION
**Pull request by Jkrzy**
## Description

Small tweak for https://github.com/18F/tock/issues/733#issuecomment-370942684

To create a development admin account, users should call ``createsuperuser`` with a non-email username.

Manually inserting an email username breaks ``utilization.GroupUtilizationView.get_queryset`` as email addresses are not caught by the current regex for the ``reports:ReportingPeriodUserDetailView`` urlpattern.

## Additional information

Not expected to encounter non-conforming usernames outside of development as ``@DOMAIN`` is truncated on user creation. There's only a single authorized domain at the moment,  this approach may lead to issues down the line if/when others are added.

### CODE REVIEW

**Comment by rogeruiz** [comment]
This looks good. I’m curious why we need to pass a username flag at all. An email should be all we need and then have the app handle the username handle for us. Wdyt?

Also `admin.user@gsa.gov` would help us ensure that usernames are being created appropriately too.

**Comment by Jkrzy** [comment]
We're using the [Django provided](https://docs.djangoproject.com/en/1.11/ref/django-admin/#createsuperuser) ``createsuperuser`` which requires both username and email. Pretty standard Django dev workflow.

We could roll our own management command, that routes through the existing user creation functionality, but there's not much to be gained from a testing perspective. The user creation path is already covered by tests.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ If your team uses Tock and Slack, you might also find the ["angrytock" reminder
   $ docker-compose build
   $ docker-compose run app python manage.py migrate
   $ docker-compose run app python manage.py loaddata test_data/data-update-deduped.json
-  $ docker-compose run app python manage.py createsuperuser --username admin@gsa.gov --email admin@gsa.gov --noinput
+  $ docker-compose run app python manage.py createsuperuser --username admin --email admin@gsa.gov --noinput
```
Comment (rogeruiz) [inline README.md]: @Jkrzy, What about making it: _new lines **just** for readability here_

```sh
docker-compose run app \
    python manage.py \
    createsuperuser \
    --username admin.user \
    --email admin.user@gsa.gov \
    --noinput
```

Comment (Jkrzy) [reply README.md]: Great thought! That would stick out as abnormal on the employees URL path.

**Pull request merged**